lib/ansible/module_utils/facts.py

```diff
@@ -99,6 +99,7 @@ class Facts(object):
                     ('/etc/os-release', 'SuSE'),
                     ('/etc/gentoo-release', 'Gentoo'),
                     ('/etc/os-release', 'Debian'),
+                    ('/etc/debian_version', 'Debian'),
```

Comment:
Right -- but is it the same format as os-release?  If it's not the same format you'll need to add different parsing rules for that file below.